config/bmo/kustomization.yaml

```diff
@@ -13,11 +14,13 @@ configMapGenerator:
   - DEPLOY_RAMDISK_URL=${DEPLOY_RAMDISK_URL}
   - IRONIC_ENDPOINT=${IRONIC_URL}
   - IRONIC_INSPECTOR_ENDPOINT=${IRONIC_INSPECTOR_URL}
+  - IRONIC_CACERT_FILE=/opt/metal3/certs/ca/tls.crt
```

Comment:
Can't we parameterize it as any other vars here?

Reply:
No this is within the pod. What matters is the secret we mount there. And we create it based on the Base64 of the certificate